Courir/Courir/MenuViewController.swift

```diff
@@ -10,13 +10,21 @@ import UIKit
 import SpriteKit
 
 class MenuViewController: UIViewController {
-
+    private var nameTextField: UITextField?
```

Comment:
Do you have to save the text field and save action as instance variables? Can we just the variables into your validation functions? It seems kind of weird.